### PR TITLE
[DA-2115/2116] Updating AW0 and AW1 manifests

### PR DESCRIPTION
**Pull request by dtharpeuno**
## Resolves *[ticket DA-2115/2116]*
https://precisionmedicineinitiative.atlassian.net/browse/DA-2115
https://precisionmedicineinitiative.atlassian.net/browse/DA-2116

## Description of changes/additions
Based on meetings with the higher-order genomic counsel, there are requests for updates to the AW0 and AW1 manifests.
AW0 - remove columns, **converted raw SQL to alchemy dao method
AW1 - new columns added to be expected for ingestion, added save to AW1 raw model, will backfill column once the PR is on prod

## Tests
- [x] unit tests

### CODE REVIEW

File: rdr_service/dao/genomics_dao.py [hunk 1]
```diff
@@ -740,6 +740,44 @@ def get_control_sample_for_gc_and_genome_type(self, _site, genome_type, biobank_
...
+    def get_new_participants(
+        self,
+        genomic_workflow_state,
+        biobank_prefix=None,
+        genomic_set_id=None,
+        is_sql=False
+    ):
+        with self.session() as session:
+            participants = session.query(
+                sqlalchemy.bindparam('value', ''),
```
Comment (j-kanuch) [inline rdr_service/dao/genomics_dao.py]: Do they want to remove `value` as well?

Comment (dtharpeuno) [reply rdr_service/dao/genomics_dao.py]: Your right, https://github.com/all-of-us/raw-data-repository/pull/2534/commits/afd7e5ab0117857f482de67a3dcbe6d2dbe69c81, edited

**Review by j-kanuch — APPROVED** [review]

File: tests/cron_job_tests/test_genomic_pipeline.py [hunk 2]
```diff
@@ -169,6 +188,7 @@ def _write_cloud_csv(file_name, contents_str, bucket=None, folder=None):
         n = clock.CLOCK.now()
         ntime = time.mktime(n.timetuple())
         os.utime(provider.get_local_path(path), (ntime, ntime))
+        return cloud_file
```
Comment (kenny-skaggs) [inline tests/cron_job_tests/test_genomic_pipeline.py]: Where is this return used? And does this work? I'm not sure of how the scoping works, but I thought the file wouldn't be usable outside of the scope of the context manager that opens it.

Comment (dtharpeuno) [reply tests/cron_job_tests/test_genomic_pipeline.py]: Yea it does work. Just used for getting the local RAW path of the CSV which is written to something like `var/tmp` , for verifying the physical columns/row(s) of the local CSV that is outputted.

Screen Shot 2021-07-27 at 12 39 32 PM

**Review by kenny-skaggs — APPROVED** [review]

**Pull request closed without merging**